### ISSUE TITLE
update requirements doc to use git 1.8.4

### ISSUE THREAD

**Issue by logicminds** [samrocketman/gitlab-mirrors#80]
The requirements say git 1.6.5 but I have git 1.7.1 on my system and I am getting errors with the update mirrors functionality.  The error is that whenever prune is called only newer versions of git have this.

I don't know which version but I assume > 1.8.4.  I have tested on 2.4.8 and it seems to work.

**Comment by samrocketman** [comment]
Seems [`git prune` was first introduced in 2006](https://github.com/git/git/commit/ba84a797e76c27932e0b317c7ce54925e81093f7).  After investigation `git prune` was [available in git 1.4.2](https://github.com/git/git/tree/v1.4.2).  I doubt the root of the issue is around `git prune` not being available.  Can you perhaps elaborate on the error you see?

**Comment by logicminds** [comment]
Here is the error that is performed when doing an update.

```
-bash-4.1$ more /home/gitmirror/gitlab-mirrors/cron.log
error: unknown option `prune'
usage: git push [<options>] [<repository> [<refspec>...]]

    -v, --verbose         be more verbose
    -q, --quiet           be more quiet
    --repo <repository>   repository
    --all                 push all refs
    --mirror              mirror all refs
    --delete              delete refs
    --tags                push tags (can't be used with --all or --mirror)
    -n, --dry-run         dry run
    --porcelain           machine-readable output
    -f, --force           force updates
    --thin                use thin pack
    --receive-pack <receive-pack>
                          receive pack program
    --exec <receive-pack>
                          receive pack program
    -u, --set-upstream    set upstream for git pull/status
    --progress            force progress reporting

error: unknown option `prune'



```

**Comment by samrocketman** [comment]
Ah, I get it.  `git push --prune` is the option and that wasn't introduced until v1.8.0.  Agreed, the documentation should be updated.

**Comment by logicminds** [comment]
For anybody seeing this error and cannot update to a newer version of git you can disable the prune feature in the gitlab-mirrors config file.

**Comment by samrocketman** [comment]
Just released v0.5.2 which is certified with GitLab 8.1.4.  `master` reflects the latest stable release.
